python/repo/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                repo
-epoch               20160202
```

Comment:
not sure about removing it. Is it required?

Reply:
A port's epoch can never be decreased (or removed, which decreases it to 0). https://guide.macports.org/#reference.keywords